### ISSUE TITLE
t/305-APNIC.t started to fail

### ISSUE THREAD

**Issue by eserte** [arhuman/Net-Whois-RIPE#22]
On my smoker systems:
```
Unknown method 'last_modified' for object Net::Whois::Object::Mntner::APNIC (Did the Database schema changed ?) at /home/cpansand/.cpan/build/2018012021/Net-Whois-RIPE-2.006002-0/blib/lib/Net/Whois/Generic.pm line 552.

#   Failed test 'Client performs queries without dying Can't require Net::Whois::Object::Mntner::IDNIC (No such file or directory) at /home/cpansand/.cpan/build/2018012021/Net-Whois-RIPE-2.006002-0/blib/lib/Net/Whois/Object.pm line 693, <GEN0> line 7.
# '
#   at t/305-APNIC.t line 17.

#   Failed test 'undef isa 'Net::Whois::Object::Mntner::APNIC''
#   at t/305-APNIC.t line 18.
#     undef isn't defined
Unknown method 'last_modified' for object Net::Whois::Object::InetNum::APNIC (Did the Database schema changed ?) at /home/cpansand/.cpan/build/2018012021/Net-Whois-RIPE-2.006002-0/blib/lib/Net/Whois/Generic.pm line 552.
# Looks like you failed 2 tests of 10.
t/305-APNIC.t ................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/10 subtests 
```

Also, the last test reports on CPAN Testers are (at the time of this writing) all fail reports:
http://matrix.cpantesters.org/?dist=Net-Whois-RIPE%202.006002;reports=1#sl=7,1

**Comment by arhuman** [comment]
Thank you very much.
I'll try to fix that asap.

Le 21 janv. 2018 11:41 PM, "Slaven Rezić" <notifications@github.com> a
écrit :

**Comment by arhuman** [comment]
Fixed in 2.007000
